transfluent.py

```diff
@@ -230,12 +230,15 @@ def file_save(self, identifier, language, file, type, format='UTF-8',
         try:
             content = file.read()
         except AttributeError:
-            content = str(file)
+            try:
+                content = bytes(file, format)
+            except TypeError:
```

Comment:
This would break in Python 3 if the user supplied the file as bytes: 

```python
>>> bytes(b'foo', 'utf-8')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: encoding or errors without a string argument
>>> str(b'foo')
"b'foo'"
```
Let's just assume that the `file` argument is either a file object or text (`unicode` in Python 2; `str` in Python 3) and replace the `try..except` with `content = file.encode(format)`.